include/bout/sundials_backports.hxx

```diff
@@ -39,7 +39,7 @@ inline void SUNNonlinSolFree(MAYBE_UNUSED(SUNNonlinearSolver solver)) {}
 #if SUNDIALS_VERSION_MAJOR < 6
 namespace sundials {
 struct Context {
-  Context(MPI_Comm comm MAYBE_UNUSED() = MPI_COMM_NULL) {}
+  Context(void* comm MAYBE_UNUSED() = MPI_COMM_NULL) {}
```

Comment:
Do we need a default argument? We probably want to make sure we always pass a valid MPI_Comm?
```suggestion
  Context(void* comm MAYBE_UNUSED()) {}
```

Reply:
Yes, the default argument can go